huaweicloud/services/acceptance/dc/resource_huaweicloud_dc_virtual_interface_test.go

```diff
@@ -107,6 +107,60 @@ func TestAccVirtualInterface_basic(t *testing.T) {
 	})
 }
 
+func TestAccVirtualInterface_acrossTenant(t *testing.T) {
+	var (
+		vif interfaces.VirtualInterface
+
+		rName = "huaweicloud_dc_virtual_interface.test"
+		name  = acceptance.RandomAccResourceName()
+		vlan  = acctest.RandIntRange(1, 3999)
+	)
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&vif,
+		getVirtualInterfaceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckDcDirectConnection(t)
+			acceptance.TestAccPreCheckDcHostedConnection(t)
+			acceptance.TestAccPreCheckTargetTenantDcVGW(t)
```

Comment:
Please check the pre-detection conditions used. Some conditions are unnecessary and need to be removed.